### PR TITLE
ref #30 from 1 to 100

### PR DESCRIPTION
**Pull request by hoangcongminh901**

### CODE REVIEW

File: IT.csv [hunk 1]
```diff
@@ -1,104 +1,104 @@
...
+34,処理する,しょりする,xử lý,process
+35,コード,こーど,code,code
+36,アプリケーション,あぷりけーしょん,application,application
+37,インストールする,いんすとーるする,cài đặt vào,install
+38,全て,すべて,toàn bộ,all
+39,ソフトウェア,そふとうぇあ,phần mềm,software
+40,技術,ぎじゅつ,kỹ thuật,technology
+41,公開する,こうかいする,công bố(ra mắt),expose(publish)
+42,追加する,ついかする,thêm vào,add
+43,認証する,にんしょうする,chứng minh là xác thực,authenticate
```
Comment (bs90) [inline IT.csv]: chứng thực?

**Comment by johnny-miyake** [comment]
LGTM!
[LGTM](http://www.lgtm.in/i/AEH1Dwqbq)

File: IT.csv (hunk 1 above)
Comment (hoangcongminh901) [inline IT.csv]: @bs90 azas

**Comment by j-miyake** [comment]
LGTM!
[LGTM](http://www.lgtm.in/i/AEH1Dwqbq)

**Pull request merged**